pyrender/mesh.py

```diff
@@ -305,9 +308,15 @@ def _get_trimesh_props(mesh, smooth=False):
                     alphaCutoff=mat.alphaCutoff
                 )
             elif isinstance(mat, trimesh.visual.texture.SimpleMaterial):
```

Comment:
Hey, I recently added a [SimpleMaterial.to_pbr](https://github.com/mikedh/trimesh/blob/507d066d7205a4c2917ba5f1d4cc34c57a2e7554/trimesh/visual/material.py#L54-L59) method that might be useful for this too, so `GLB` exports of `OBJ` imports look nice. That glossiness/roughness conversion looks great, is there a reference for that somewhere? I'd love to add better logic to `to_pbr`.